app/controllers/new_gestionnaire/procedures_controller.rb

```diff
@@ -121,6 +119,10 @@ def update_sort
 
     private
 
+    def statut
+      @statut = params[:statut].present? ? params[:statut] : 'a-suivre'
```

Comment:
pourquoi le mettre en variable d'instance ? je serai chaud pour faire sauter l'affectation

Reply:
En fait j'ai oublié la mémoization, ça a plus de sens dans le commit updaté

Et on a besoin de le mettre en variable d'instance car c'est utilisé dans le template

Reply:
Du coup, le nom m'embete car il n'indique pas qu'on set la variable ...
`def set_status` ?

Reply:
Oui, pareil avec `procedure_presentation`, `procedure_sort`, `procedure_filters`, `current_filters`.

Il me semble que c'est standard en Rails, cf. https://www.justinweiss.com/articles/4-simple-memoization-patterns-in-ruby-and-one-gem/